packages/client/src/components/ToolItem/ToolItem.component.js

```diff
@@ -120,13 +121,30 @@ export const ToolItem = ({ tool }) => {
         </div>
       </div>
 
-      <Link to={`/tools/${tool.id}`}>
+      {localUser && localUser.is_admin ? (
         <div className="button-container">
           <button type="button" className="tool-button">
```

Comment:
We don't need to repeat the View Tool button here. Have you tried only wrapping the edit and delete links in this condition?